app/forms/support_user_invite_form.rb

```diff
@@ -0,0 +1,52 @@
+class SupportUserInviteForm
```

Comment:
Could you adopt the usage of the `ApplicationForm` for new form objects?

Reply:
Ref: https://github.com/DFE-Digital/itt-mentor-services/blob/main/docs/conventions/form_objects.md

Reply:
resolved in [e51e970](https://github.com/DFE-Digital/itt-mentor-services/pull/253/commits/e51e970a5c60e4d2118cbf6f50ff8310c78d2e67)